core/config/initializers/00_configuration.rb

```diff
@@ -179,4 +179,18 @@
       },
       description: 'Subjects list for the contact form'
   end
+
+  fieldset 'Security', namespaced: false do
+    field 'Safe IP Addresses',
+      id: :safe_ip_addresses,
+      type: :array,
+      default: [],
+      description: 'List of known IP addresses that should always have access to the site.'
```

Comment:
Would you change the language or add a tooltip or something to clarify that they can specify range with the mask?